src/status_im/common/floating_button_page/view.cljs

```diff
@@ -52,7 +52,8 @@
      :remove-listeners    remove-listeners}))
 
 (defn view
-  [{:keys [header footer customization-color footer-container-padding gradient-cover?]
+  [{:keys [header footer customization-color footer-container-padding header-container-padding
```

Comment:
maybe actually - header-container-style and we pass  a map with padding in that?
and probably should do the same for footer (but we do this in follow up)

Reply:
done 👍